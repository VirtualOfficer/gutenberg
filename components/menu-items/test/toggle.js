/**
 * External dependencies
 */
import { shallow } from 'enzyme';

/**
 * Internal dependencies
 */
import MenuItemsToggle from '../toggle';

describe( 'MenuItemsToggle', () => {
	test( 'should match snapshot when only label provided', () => {
		const wrapper = shallow(
			<MenuItemsToggle
				label="My item"
			/>
		);

		expect( wrapper ).toMatchSnapshot();
	} );

	test( 'should match snapshot when all props provided', () => {
		const wrapper = shallow(
			<MenuItemsToggle
				isSelected={ true }
				label="My item"
				onClick={ () => {} }
				shortcut="mod+shift+alt+w"
			/>
		);

		expect( wrapper ).toMatchSnapshot();
	} );
} );
