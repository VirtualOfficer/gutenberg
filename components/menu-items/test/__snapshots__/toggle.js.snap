// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuItemsToggle should match snapshot when all props provided 1`] = `
<MenuItemsItem
  className="is-toggle"
  icon="yes"
  isSelected={true}
  label="My item"
  onClick={[Function]}
  shortcut="mod+shift+alt+w"
/>
`;

exports[`MenuItemsToggle should match snapshot when only label provided 1`] = `
<MenuItemsItem
  className="is-toggle"
  label="My item"
/>
`;
